src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -226,10 +226,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
                 messageText+= imageText.getDescription() + ", ";
               }
             }
-			System.out.println("************************ Text detected: "+messageText+"*************************");
             
-           if(messageText.toLowerCase().contains("donated") || messageText.toLowerCase().contains("gave, blood") || messageText.toLowerCase().contains("donor") || messageText.toLowerCase().contains("donation")) 
-           {
+           if(containsKeyword(messageText, "donated", "gave, blood", "donor", "donation"))
+		   {
                 if(messageText.toLowerCase().contains("blood"))
                     messageText= messageUrl + "Thanks for donating blood! you can save a life!";
                 else if(messageText.toLowerCase().contains("money") || messageText.toLowerCase().contains("dollar") || messageText.toLowerCase().contains("$"))
```

Comment:
You can also use containsKeywords on this line as well

Reply:
Yes! you're right!